src/app/tic-tac-toe.component.ts

```diff
@@ -79,7 +79,7 @@ export class TicTacToeComponent {
       } else if (this.playerTurn === Turn.Second) {
         this.cells[i][j] = Cell.Circle;
         this.playerTurn = Turn.First;
-      }
+      } // review
```

Comment:
The `Turn` and `Cell` enums are basically synonymous. Using just one of them for both players and cells would spare you unnecessary mapping throughout the code.